src/utils/AuthServices/AuthServices.js

```diff
@@ -0,0 +1,34 @@
+import decode from "jwt-decode";
+
+let token = localStorage.getItem("accessToken");
+
+const setToken = (token) => {
+  localStorage.setItem("accessToken", token);
+};
+
+const isTokenExpired = (token) => {
+  try {
+    const decoded = decode(token);
+    if (decoded.exp < Date.now() / 1000) {
+      return true;
+    } else return false;
```

Comment:
Nie osłabiaj mnie xd

```suggestion
    return decoded.exp < (Date.now() / 1000)
```